docs/sources/writing-guide/contribute-release-notes/index.md

```diff
@@ -12,6 +12,127 @@ keywords:
+  - This change is invisible to customers.
+
+## What's New doc development process
+
+Developing directly in Markdown reduces errors and removes a manual (toilsome and error-prone step), by creating, editing, discussing, and publishing in the same format from which we will publish in GitHub, using the Grafana repo. 
+
+We also recommend that you think critically about whether to include an image in the What’s New document. We will not maintain these images, as the document refelct a "point in time". If you need to add an image, refer to [Image, diagram, and screenshot guidelines]({{</* relref "../image-guidelines" */>}}).
+
+### When the prior release is complete
+
+Complete the following steps after the prior release goes out.
+
+1. The Docs Squad identifies the technical writer responsible for overseeing the What’s New content development process for the upcoming release.
+1. The identified technical writer cuts a branch and creeates a PR with an empty What's New doc that will be populated with the What’s New content for the next release. 
+1. The technical writer then pins a link to [version-number-release-gtm](https://raintank-corp.slack.com/archives/C02L8GQJGBW) Slack channel.
+
+   > **Note:** A Product Manager (PM) is responsible for update the version number of that channel after a release goes out. 
+   
+1. The technical writer posts in the Slack channel that the What's New PR is ready for populating. 
+1. The PM and Product Marketing Manager (PMM) organizations identify which PM or PMM is responsible for vetting content added to the What's New draft.
+1. The PM or PMM on point communicates with eng squads without a PM resource about who’s on point for What’s New PR reviews from within the PM and PMM orgs.
+
+### Throughout a release
+
+A Grafana release cycle is typically 6-8 weeks, and during that time, developers are empowered to add to the What's New document incrementally. To see a list of release dates and assigned PMs and PMMs, refer to [Grafana Releases and Release Dates](https://grafana-intranet--simpplr.visualforce.com/apex/simpplr__app?u=/site/a145f000001dCXBAA2/page/a125f000001AXrwAAG).
+
+During this time, the role of the technical writer is to periodically check in on the What's New PR to determine how much content has been added.
+
+Complete the following steps throughout a release cycle:
+
+1. Dev teams cut PRs against the branch and add What’s New content.
+   - If a dev team has a PM, the dev team adds that PM or PMM as Reviewer.
+   - If a dev team has a writer, the dev teams add the writer as Reviewer.
+   - If a dev team does not have a PM or PMM, the dev team adds the identified PM or PMM as the reviewer.
+1. Contributors label PRs with `add to what’s new` and write the PR description so it can be used as draft source for the What's New document. 
+
+   > **Note:** This won't need to apply to every PR - just the primary PR that checks in the new feature, changed feature, or deprecation. 
```

Comment:
@mjseaman , I pulled in this Note from the gdoc guidelines, but I don't understand what it means. Do you remember?

Reply:
It's a label we are using so we can track which content needs to be added to what's new. I think it's just the alerting team using it--but it works a treat!

Reply:
Just to clarify - devs should add the what's new label to the PR they cut against the branch? I'm confused by "This won't need to apply to every PR - just the primary PR..." bit. I consider the 'primary PR' to be the parent PR that eventually gets merged into main.